### PR TITLE
Add EOmaps package

### PR DESCRIPTION
**Pull request by raphaelquast**
## PR Summary

I've just been reading through the doc and thought it might be a good idea to add my package to the list...

### CODE REVIEW

**Comment by story645** [comment]
Your package looks so cool, thanks for adding it!

File: packages/eomaps.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+repo: raphaelquast/eomaps
+section: mapping
+description: A library to create interactive maps of geographical datasets
+site: https://github.com/raphaelquast/EOmaps
+keywords:  interactive mapping
```
Comment (story645) [inline packages/eomaps.yml:5]: This should be a list

**Review by story645 — CHANGES_REQUESTED** [review]

File: packages/eomaps.yml (hunk 1 above)
Comment (raphaelquast) [reply packages/eomaps.yml]: should be fixed

File: packages/eomaps.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+repo: raphaelquast/eomaps
+section: mapping
+description: A library to create interactive maps of geographical datasets
+site: https://github.com/raphaelquast/EOmaps
```
Comment (jklymak) [inline packages/eomaps.yml:4]:
```suggestion
site: https://raphaelquast.github.io/EOmaps/ 
```
?  We already link the repo...

**Comment by jklymak** [comment]
No, we automatically detect the badges, and we don't deal with those other ones.  (ahem, left over from the project where we copied this ;-)

**Comment by jklymak** [comment]
I squashed when I merged, just in case you are tracking commits for some reason ;-) Thanks for the addition!

**Comment by raphaelquast** [comment]
awesome, thanks for merging!

**Pull request merged**